src/Request/uapi-parser.js

```diff
@@ -258,6 +284,7 @@ Parser.prototype.parse = function (xml) {
       if (self.debug > 2) {
         console.log('Error during automatic resolving version');
       }
+      throw new RequestRuntimeError.VersionParsingError(obj, e);
```

Comment:
I'd like to keep it too!
I meant removing these lines:
```javascript
       if (self.debug > 2) {
         console.log('Error during automatic resolving version');
       }
```

As we don't need logging if we are throwing an error